### ISSUE TITLE
v3.0.0 import path major version mismatch

### ISSUE THREAD

**Issue by System-Glitch** [prebid/prebid-server#4030]
It is currently impossible to import version v3.0.0 of the package because of mismatch with the import path. 

```
$ go get github.com/prebid/prebid-server/v2@v3.0.0
go: github.com/prebid/prebid-server/v2@v3.0.0: invalid version: module path includes a major version suffix, so major version must match
```

The go toolchain enforces the major version number to match with the import path. I suggest the following changes:

go.mod:
```diff
- module github.com/prebid/prebid-server/v2
+ module github.com/prebid/prebid-server/v3
```

Everywhere else:
```diff
- github.com/prebid/prebid-server/v2
+ github.com/prebid/prebid-server/v3
```

You then need to invalidate the current v3.0.0 tag by editing the `go.mod`:
```
retract v3.0.0 // Forgot to update major version in import path and module name
```

Go docs : 
- https://go.dev/doc/modules/version-numbers
- https://go.dev/doc/modules/major-version 
- https://go.dev/doc/modules/gomod-ref#retract

**Comment by System-Glitch** [comment]
I just noticed [this MR](https://github.com/prebid/prebid-server/pull/4029) that is fixing the issue already.

**Comment by SyntaxNode** [comment]
We realized this just after cutting the release. v3.0.1 will be released today with the fix. I've added the retract statement to fix the import toolchain.

Impressed you noticed this so quickly. Is this a bot?

**Comment by System-Glitch** [comment]
We have a notification for new prebid-server releases. While trying to upgrade this morning I noticed the issue.

**Comment by SyntaxNode** [comment]
Addressed in v3.1.0.
